### PR TITLE
Revise uwsgi defaults and display.

### PR DESCRIPTION
**Pull request by jmchilton**
- In https://github.com/galaxyproject/galaxy/issues/1530 we realized that buffer-size needs to be increased for Galaxy requests - so increase the default.
- Mention thunder-lock since that should frequently be enabled - https://github.com/galaxyproject/ansible-galaxy-extras/pull/195.

### CODE REVIEW

File: lib/galaxy/webapps/config_manage.py [hunk 1]
```diff
@@ -106,6 +106,11 @@
         'default': '$uwsgi_module',
         'type': 'str',
     }),
+    ('thunder-lock', {
+        'desc': """It is usually a good idea to set this to ``true`` if processes is greater than 1.""",
+        'default': 'false',
+        'type': 'str',
```
Comment (nsoranzo) [inline lib/galaxy/webapps/config_manage.py]: These could be changed to:
```
        'default': False,
        'type': 'bool',
```
after https://github.com/galaxyproject/galaxy/pull/5747 is merged.

File: scripts/get_uwsgi_args.py [hunk 2]
```diff
@@ -89,6 +89,7 @@ def _get_uwsgi_args(cliargs, kwargs):
         'virtualenv': os.environ.get('VIRTUAL_ENV', './.venv'),
         'pythonpath': 'lib',
         'threads': '4',
+        'buffer-size': '16384',  # https://github.com/galaxyproject/galaxy/issues/1530
```
Comment (nsoranzo) [inline scripts/get_uwsgi_args.py]: This needs to be added also at line 26 above.

**Comment by jmchilton** [comment]
@nsoranzo Rebased on top of fixes from #5747. I also think I fixed the other issues you found.

**Review by nsoranzo — APPROVED** [review]

**Comment by natefoo** [comment]
I think this needs to be backported to 18.01? We also need to update the scaling docs.

**Pull request merged**